ProjectMars/Features/Language.feature

```diff
@@ -0,0 +1,38 @@
+Feature: LangFeature
+
+Scenario: A.Add a Language To User Profile
+Given User Able Login Into MarsPortal Application Successfully
+When Create a NewLanguage Into User Profile '<Language>' and '<Level>'
+Then The NewLanguage Record Created '<Language>' and '<Level>' successfully created
+
+Examples: 
```

Comment:
so you are missing test cases you have not thought about 
• Negative testing with valid input
• Negative testing with invalid input
• Destructive testing

E.g of above scenarios
Happy path tests check basic functionality and the acceptance criteria of the API. We later extend positive tests to include optional parameters and extra functionality.

The next group of tests is negative testing where we expect the application to gracefully handle problem scenarios with both valid user input (for example, trying to add an existing username) and invalid user input (trying to add a username which is null).

Destructive testing is a deeper form of negative testing where we intentionally attempt to break the API to check its robustness (for example, sending a huge payload body in an attempt to overflow the system).


e.g what happens if you add a language that exists, deleting  language that does not exist etc!

please increase the coverage!